### DISCUSSION TITLE
I have a route matching question.

### DISCUSSION THREAD

**Discussion by cxytz01** [Kong/kong#7572] [Help] [answered]
Hello, it is my first time using Kong gateway.

I have a route matching question.
- Kong version: 2.1.3

I have some REST URI like these:

1. POST  [/api/v1/user ](URL)
2. GET    [/api/v1/user/{:userid}/verify?code={code}](URL)
3. GET    [/api/v1/user/{:userid}/exist  ](URL)
4. POST [/api/v1/user/{:userid}/password/reset](URL)
5. GET    [/api/v1/user/{:userid} ](URL)

I want to 1,2,3,4 cases route to upstream A, and 5 cases route to upstream B.

What should I do with the rule of the path?

I have read this document: https://docs.konghq.com/gateway-oss/2.1.x/proxy/#request-path, and have no idea to write the rules and regex_priority.

**Comment by khotaditya** [comment, marked as answer]
Hello,

First thing, you've to create 2 services with different upstreams
For example:
Service A with upstream A and Service B with upstream B
Then you've to create routes with different host header to access those routes.
For example:
In Service A create routes 1, 2, 3, 4 using host header "example_1.com"
In Service B create route 5 using host header "example_2.com"
